python_modules/dagster/dagster/_core/instance/__init__.py

```diff
@@ -830,6 +830,9 @@ def get_settings(self, settings_key: str) -> Any:
             return self._settings.get(settings_key)
         return {}
 
+    def get_backfill_settings(self) -> Mapping[str, Any]:
```

Comment:
We have overrides for these `get_X_settings` in internal [here](https://github.com/dagster-io/internal/blob/f004579a15f1f7d694780c28452558ba9875af08/dagster-cloud/python_modules/dagster-cloud-backend/dagster_cloud_backend/instance.py#L502) and I think you'll want to do the same for the backfill settings. @gibsondan please correct me if that's not necessary in this case

Reply:
I'd created an internal PR when I first raised this! https://github.com/dagster-io/internal/pull/12871

Not sure what the exact procedure is (does the OSS PR go in before the internal one?).

Reply:
typically you would land them at the same time - as long as they go out in the same release it should be fine.